serverless.yml

```diff
@@ -28,6 +28,7 @@ provider:
     REDIS_PORT: ${env:REDIS_PORT}
     REDIS_DB: ${env:REDIS_DB}
     TOKEN_METADATA_BUCKET: ${env:TOKEN_METADATA_BUCKET}
+    CORS_ALLOWED_ORIGIN: ${env:CORS_ALLOWED_ORIGIN}
```

Comment:
We have to set this new variable in `.github/workflows/deploy.yml`

Reply:
Done